packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -58,16 +67,12 @@ v0.31.0
 
 ### Changed
 - Updated skipped tests to work with Storybook component tests.
+- Updated config for latest `sass-loader`.
```

Comment:
Was this moved because the changes were actually made on the 5th?

Reply:
@DamianMullins I moved it because we don't seem to have a convention of having multiple dates for a single version